apps/src/templates/instructions/codeReview/CommentEditor.jsx

```diff
@@ -6,14 +6,53 @@ import Button from '@cdo/apps/templates/Button';
 import color from '@cdo/apps/util/color';
 
 export default class CommentEditor extends Component {
-  static propTypes = {onNewCommentSubmit: PropTypes.func.isRequired};
+  static propTypes = {
+    onNewCommentSubmit: PropTypes.func.isRequired,
+    onNewCommentCancel: PropTypes.func.isRequired,
+    saveError: PropTypes.bool,
+    saveInProgress: PropTypes.bool
+  };
 
   state = {comment: ''};
 
   commentChanged = event => this.setState({comment: event.target.value});
 
+  renderSaveStatus() {
+    const {saveError, saveInProgress} = this.props;
+    let icon = '';
+    let saveMessageTitle = '';
+    let saveMessageText = '';
+
+    if (saveInProgress) {
+      icon = <span className="fa fa-spin fa-spinner" style={styles.spinner} />;
```

Comment:
Could we just reuse the `<Spinner/>` component here?

Reply:
done!